sparse_ho/criterion/multiclass_logreg.py

```diff
@@ -15,12 +15,15 @@ class LogisticMulticlass():
         indices of the training set
     idx_val: np.array
         indices of the validation set
-    algo: TODO
-    idx_test: TODO
+    algo: instance of ``sparse_ho.base.AlgoModel``
+        A model that follows the sparse_ho API.
+    idx_test: np.array
```

Comment:
```suggestion
    idx_test: ndarray
```

array is not the type in numpy. It's ndarray. I can live with np.ndarray or just ndarray